tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -282,8 +290,14 @@ EOF
 	# that does not exist.
 	#
 	echo "::group::Check the service healthy"
-	kbs_ip=$(kubectl get -o jsonpath='{.spec.clusterIP}' svc "$KBS_SVC_NAME" -n "$KBS_NS" 2>/dev/null)
-	kbs_port=$(kubectl get -o jsonpath='{.spec.ports[0].port}' svc "$KBS_SVC_NAME" -n "$KBS_NS" 2>/dev/null)
```

Comment:
it'd be better to utilize the `kbs_k8s_svc_http_addr` function (or kbs_k8s_svc_host and kbs_k8s_svc_port`)

Also those functions should be modified to reflect the changes.

Reply:
changes applied